cdc/log.cc

```diff
@@ -820,7 +838,7 @@ class transformer final {
             const mutation& m)
     {
         auto& p = m.partition();
-        if (p.partition_tombstone() || !p.row_tombstones().empty() || p.clustered_rows().empty()) {
+        if (p.clustered_rows().empty() && p.static_row().empty()) {
```

Comment:
Why is this good? I think we shouldn't generate preimage for partition deletion.